beets/util/__init__.py

```diff
@@ -794,9 +798,15 @@ def convert(arg):
     return [convert(a) for a in args]
 
 
+# stdout and stderr as bytes
+CommandOutput = namedtuple("CommandOutput", ("stdout", "stderr"))
```

Comment:
I don't know much about ffmpeg, etc -- but if you have the patience to split up this `CommandOutput` change into a separate PR it feels pretty low risk so I'll happily sign off so we can at least get part of this merged and make the rest easier to review?

Reply:
That is a great suggestion, I'll get right to it!

Reply:
Done with  #3329.